Smarty/templates/modules/Documents/pdfViewer.html

```diff
@@ -21,12 +22,35 @@
       #next::before, #previous::before {
           display: contents;
       }
+      #savingProgress::before{
+        background: none;
+      }
+      #dragSignature {
+        touch-action: none;
+        user-select: none;
+        transform: translate(50%, 90%);
+        z-index: 100000;
+        position: absolute;
+        border: 2px solid #000;
+        width: 10%;
+      }
+      #dragSignatureCtrBtn {
+        position: relative;
+        top: -8px;
+      }
     </style>
+    <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.11.6/dist/umd/popper.min.js" integrity="sha384-oBqDVmMz9ATKxIep9tiCxS/Z9fNfEXiDAYTujMAeBAsjFuCZSmKbSSUnQlmh/jp3" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/js/bootstrap.min.js" integrity="sha384-IDwe1+LCz02ROU9k972gdyvl+AESN10+x7tBKgc9I5HFtuNz0wWnPclzo6p9vxnk" crossorigin="anonymous"></script>
```

Comment:
same as the previous comment, and also, why do we need to have dynamic downloads? I don't like having external dependencies that change on us whenever somebody else wants.